### PR TITLE
Fix memory leak issue

### PR DESCRIPTION
**Pull request by DmitriyMolch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ## vNext
 - ...
 
+## v0.24.2
+- [FE] Fix memory leak issue
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md]: It is not FE. It is BE.

File: api/src/services/ethNetworkService.js [hunk 2]
```diff
@@ -54,10 +55,18 @@ module.exports = {
...
+    
+    const contract = 
+      contractCreated ?
+      contracts.get(contractKey) :
+      new web3.eth.Contract(
+        erc20TokenAbi,
+        erc20TokenAddress
+      );
+
+    if(!contractCreated) contracts.set(contractKey, contract);
```
Comment (AndreiNavoichyk) [inline api/src/services/ethNetworkService.js]: Please, format the code.

File: CHANGELOG.md (hunk 1 above)
Comment (DmitriyMolch) [reply CHANGELOG.md]: fixed

File: api/src/services/ethNetworkService.js (hunk 2 above)
Comment (DmitriyMolch) [reply api/src/services/ethNetworkService.js]: fixed

**Review by AndreiNavoichyk — DISMISSED** [review]

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**